starter/cloudstorage/src/main/java/com/udacity/jwdnd/course1/cloudstorage/services/FileService.java

```diff
@@ -0,0 +1,140 @@
+package com.udacity.jwdnd.course1.cloudstorage.services;
+
+import com.udacity.jwdnd.course1.cloudstorage.data.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.util.StringUtils;
+import org.springframework.web.multipart.MultipartFile;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.util.List;
+
+@Service
+public class FileService {
+
+
+    @Autowired
+    private UserService userService;
+
+    private final FileMapper fileMapper;
+
+    public FileService(FileMapper fileMapper) {
+        this.fileMapper = fileMapper;
+    }
+
+    public List<File> getFiles() {
+        User user = userService.getUser(UserService.getLoggedInUsername());
+        return fileMapper.select(user.getUserId());
+    }
+
+    public Message uploadFile(FileForm file)  {
+        String filename = StringUtils.cleanPath(file.getFileUpload().getOriginalFilename());
+
+        Message message = new Message();
+
+        if (file.getFileUpload().isEmpty()) {
+            message.setError(true);
+            message.setSuccess(false);
+            message.setMsg("Failed to store empty file.");
+
+            return message;
```

Comment:
Initialize `Message` data using a constructor
```suggestion
            return new Message(true, false, "Failed to store empty file.");
```